### PR TITLE
Add new releaseline callled experimental, more few fixes

### PR DESCRIPTION
**Pull request by olblak**
* Create a new releaseline called 'experimental' to not impact the current release process
* Only define BUILDENV, CREDENTIALS and BRAND variable if they are not set
* Refactor msi publishing script to read variable definition from makefile variable files

### CODE REVIEW

File: env/test.mk [hunk 1]
```diff
@@ -13,8 +13,9 @@ export SCP_OPTS=
 # where to put binary files
 export TESTDIR=$(realpath .)/pkg.jenkins.io
 export WARDIR=/packages/binary/war${RELEASELINE}
+# Concat MSDIR and RELEASELINE in the msi publishing
 export MSIDIR=/packages/binary/windows${RELEASELINE}
-export OSXDIR=export OSXDIR=${TESTDIR}/osx${RELEASELINE}
+export OSXDIR=/packages/osx${RELEASELINE}
```
Comment (timja) [inline env/test.mk]: is OSX still needed here? I thought we didn't ship it anymore

File: env/release.mk [hunk 2]
```diff
@@ -12,7 +12,7 @@ export SCP_OPTS=-P 22
 
 # where to put binary files
 export WARDIR=/packages/binary/war${RELEASELINE}
-export MSIDIR=C:\packages\binary\windows${RELEASELINE}
+export MSIDIR=/packages/binary/windows${RELEASELINE}
```
Comment (timja) [inline env/release.mk]: no longer on c:/ ?

File: env/test.mk (hunk 1 above)
Comment (olblak) [reply env/test.mk]: We don't ship it anymore and we could problaby remove those variables

File: env/release.mk (hunk 2 above)
Comment (olblak) [reply env/release.mk]: It appears that powershell is doing the translatation, at least it's working :p.

**Comment by olblak** [comment]
I propose to do the OSX code cleanup in a different pull request as It would require some additionnal testing to be sure that part of it is not used somewhere else

**Pull request merged**